aurorastation.dme

```diff
@@ -1254,6 +1254,7 @@
 #include "code\modules\background\religion\tajara.dm"
 #include "code\modules\background\religion\unathi.dm"
 #include "code\modules\background\religion\vaurca.dm"
+#include "code\modules\background\religion\diona.dm"
```

Comment:
Looks good, but please put the #include for diona.dm above the other religion includes (in alphabetical order) or else compilation and bug torture will ensue

Reply:
Should be fixed now.

Reply:
Should /actually/ be fixed after I had a small brain moment